include/tsid/measured-forces/measured-force-3Dforce.hpp

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright (c) 2022 CNRS INRIA LORIA
+//
+// This file is part of tsid
+// tsid is free software: you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation, either version
+// 3 of the License, or (at your option) any later version.
+// tsid is distributed in the hope that it will be
+// useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+// of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// General Lesser Public License for more details. You should have
+// received a copy of the GNU Lesser General Public License along with
+// tsid If not, see
+// <http://www.gnu.org/licenses/>.
+//
+
+#ifndef __invdyn_measured_force_3Dforce_hpp__
+#define __invdyn_measured_force_3Dforce_hpp__
+
+#include "tsid/measured-forces/measured-force-base.hpp"
+#include <pinocchio/multibody/data.hpp>
+
+namespace tsid
+{
+  namespace measuredForces
+  {
+    class MeasuredForce3Dforce : public MeasuredForceBase
```

Comment:
repeating the word "force" twice seems redundant. What about "Measured3dForce" or "MeasuredForce3D"?